app/components/layouts/chat-header.tsx

```diff
@@ -26,7 +26,7 @@ export function ChatHeader({
         <div className="flex items-center gap-4">
           <Avatar className="w-14 h-14">
             <AvatarImage src={image} />
-            {name.toLowerCase() === 'loading...' ? (
+            {name.toLowerCase() === 'loading...' || !name ? (
```

Comment:
I would restructure this conditional as:

```tsx
            {(name && (name.toLowerCase() !== 'loading...')) ? (
              <AvatarFallback className="bg-gray-300">
                {getInitials(name)}
              </AvatarFallback>
             ) : (
               <AvatarFallback className="bg-gray-300" />
             )}
```

I think `name &&...` reads a lot more idiomatically JavaScripty and is therefore easier to read, plus the extra parens help make the conditional clear